src/main/kotlin/com/petqua/common/query/PreparedStatementProxyHandler.kt

```diff
@@ -0,0 +1,27 @@
+package com.petqua.common.query
+
+import jakarta.annotation.Nonnull
+import jakarta.annotation.Nullable
+import org.aopalliance.intercept.MethodInterceptor
+import org.aopalliance.intercept.MethodInvocation
+
+private const val EXECUTE = "execute"
+
+class PreparedStatementProxyHandler(
+    private val queryInfo: QueryInfo,
+) : MethodInterceptor {
+
+    @Nullable
+    override operator fun invoke(@Nonnull invocation: MethodInvocation): Any? {
+        val method = invocation.method
+        if (method.name.contains(EXECUTE)) {
+            val startTime = System.currentTimeMillis()
+            val result = invocation.proceed()
+            val endTime = System.currentTimeMillis()
+            queryInfo.time += endTime - startTime
+            queryInfo.increaseCount()
+            return result
+        }
+        return invocation.proceed()
```

Comment:
Kotlin에서 `measureTimeMillis` dsl을 제공하는데 이걸 활용하면 좋을 것 같습니다!!
```suggestion
        if (method.name.contains(EXECUTE)) {
            return measureTimeMillis {
                invocation.proceed()
            }.also {
                queryInfo.time += it
                queryInfo.increaseCount()
            }
        }
        return invocation.proceed()
```

Reply:
이게뭐고 좋은데요?
96cf3d1c7da9e8b55f94c557752ad7669a0f7006